server/personality/personality.module.ts

```diff
@@ -13,6 +16,8 @@ import { ViewModule } from "../view/view.module";
 import { ConfigModule } from "@nestjs/config";
 import { CaptchaModule } from "../captcha/captcha.module";
 import { AbilityModule } from "../auth/ability/ability.module";
+import { personalityServiceProvider } from "./personality.provider";
+import dbConfig from "../config/db.config";
```

Comment:
issue: pull the config service inside the module instead of creating this custom db.config file

Reply:
Resolution for this problem not founded yet, FIXME comment add on dbConfig file.